package-res/resources/web/test/prompting/PromptPanelSpec.js

```diff
@@ -319,8 +319,10 @@ define([ 'dojo/number', 'dojo/i18n', 'common-ui/prompting/PromptPanel',
       });
 
       it("submit", function() {
+        spyOn(panel, "onSubmit");
         panel.submit();
         expect(panel.forceAutoSubmit).toBeFalsy();
+        expect(panel.onSubmit).toHaveBeenCalled();
       });
 
```

Comment:
I don't think I will be able to test if PromptPanel#onSubmit wasn't executed when it's null (I get the message "Error: Expected a spy, but got null."). What I will do is improve this test by adding the parameters **promptPanel** and **options** that should be passed on to the onSubmit function.